internal/pkg/template/templates/workloads/partials/cf/env-controller.yml

```
@@ -5,8 +5,10 @@ EnvControllerAction:
   Properties:
     ServiceToken: !GetAtt EnvControllerFunction.Arn
     Workload: !Ref WorkloadName
+{{- if not .UseImportedCerts}}
```

nit
```suggestion
{{- if and (not .UseImportedCerts) (.Aliases)}}
```
plus remove extra `end`